src/store/sagas/handlers/user.js

```diff
@@ -28,6 +28,7 @@ export function* handleRegisterUser(action) {
     try {
 
         yield call(requestRegisterUser.bind(null, action.userData))
+        yield put(enableButton())
```

Comment:
Is this needed?...

I feel it's safer to leave it disabled since we wouldn't be expecting them to do any other thing once they are authenticated... the page automatically reloads.... So not needed.

Reply:
Well that's true, but is there a tradeoff for doing it?

Reply:
I think it's best not to put it, since we are expecting no form of input/action from the user once the authentication process succeeds.

Reply:
Alright boss, thanks for the heads up.

Reply:
In short...not a needed updated